src/main/java/org/kohsuke/github/GHPullRequestReviewBuilder.java

```diff
@@ -103,15 +133,35 @@ public GHPullRequestReview create() throws IOException {
                 .wrapUp(pr);
     }
 
+    @JsonInclude(Include.NON_NULL)
     private static class DraftReviewComment {
         private String body;
         private String path;
-        private int position;
+        private Integer position;
+        private Integer line;
+        private Integer start_line;
+        private String side;
 
-        DraftReviewComment(String body, String path, int position) {
+        DraftReviewComment(String body, String path, Integer position) {
```

Comment:
For this constructor, the `position` is required and not allowed to be `null`, correct? 
```suggestion
        DraftReviewComment(String body, String path, int position) {
```